app/views/security_events/all.html.erb

```diff
@@ -1,5 +1,18 @@
 <h2>All Security Events</h2>
 
+<%= form_with url: security_events_search_path, method: :post do |form| %>
+  <fieldset class="usa-fieldset">
+    <legend class="usa-legend">
+      Filter Events By
+    </legend>
+
+    <%= form.label :email, 'Email', class: 'usa-label' %>
```

Comment:
s Email/Search by email address

Reply:
<img width="439" alt="Screen Shot 2021-04-15 at 11 03 08 AM" src="https://user-images.githubusercontent.com/458784/114917046-5f2abd80-9dda-11eb-82f8-6944af0714c2.png">

@juliaelman updated in f6ca637, I think that covers it?